src/app/panels/text-panel/text-panel.component.ts

```diff
@@ -87,17 +98,29 @@ export class TextPanelComponent implements OnInit, OnDestroy {
   }
 
   toggleSecondaryContent(newContent: string) {
+    this.msDescOpen = !this.msDescOpen;
     if (this.secondaryContent !== newContent) {
       this.showSecondaryContent = true;
       this.secondaryContent = newContent;
     } else {
+      if (this.msDescOpen) {
```

Comment:
In this cases use ternary operator. Write as `this.showSecondaryContent = this.msDescOpen ? true : false;`

Anyway it doesn't work. See comment above.